modules/gapi/test/gapi_gcomputation_tests.cpp

```diff
@@ -51,6 +52,41 @@ namespace opencv_test
           {
           }
       };
+
+      struct GComputationVectorMatsAsOutput: public ::testing::Test
+      {
+          cv::GMat in;
+          cv::GMat out[3];
+          cv::Mat  in_mat;
+          cv::GComputation m_c;
+          std::vector<cv::Mat> ref_mats;
+
+          GComputationVectorMatsAsOutput() : in_mat(300, 300, CV_8UC3),
+          m_c([&](){
+                      std::tie(out[0], out[1], out[2]) = cv::gapi::split3(in);
+                      return cv::GComputation({in}, {out[0], out[1], out[2]});
+                  })
```

Comment:
This is the most incorrect way to use lambda-based constructor.

It's whole idea was to keep GMat stuff/etc inside the lambda so not polluting the outside scope with extra variables.
Here you do the opposite.

Please move `in` and `out[3]` into the lambda itself. You don't need it in the class.

Reply:
Fixed